lib/components/Select2/Select.tsx

```diff
@@ -105,6 +135,10 @@ function Select<
         styles={customStyles}
         theme={selectTheme}
         inputId={selectID}
+        components={{
+          ...props.components,
+          ValueContainer: CustomValueContainer as any,
```

Comment:
if I'm not wrong, you can solve this by adding the Option type here
`CustomValueContainer<Option>`

and then on line 31:
const CustomValueContainer: FC<CustomValueContainerProps<OptionType>> = ({

Reply:
and also replace then any on line 28 with it

Reply:
Thanks for the review, I will look into it